tests/components/flo/test_config_flow.py

```diff
@@ -0,0 +1,68 @@
+"""Test the flo config flow."""
+import json
+import time
+
+from homeassistant import config_entries, setup
+from homeassistant.components.flo.const import DOMAIN
+
+from .common import TEST_EMAIL_ADDRESS, TEST_PASSWORD, TEST_TOKEN, TEST_USER_ID
+
+from tests.async_mock import patch
+
+
+async def test_form(hass, aioclient_mock_fixture):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.flo.async_setup", return_value=True
+    ) as mock_setup, patch(
+        "homeassistant.components.flo.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        result2 = await hass.config_entries.flow.async_configure(
+            result["flow_id"], {"username": TEST_USER_ID, "password": TEST_PASSWORD}
+        )
+
+        assert result2["type"] == "create_entry"
+        assert result2["title"] == "Home"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)